packages/mobile/src/store/offline-downloads/sagas/requestDownloadCollectionSaga.ts

```diff
@@ -20,6 +20,7 @@ export function* requestDownloadCollectionSaga() {
 
 function* downloadCollection(action: CollectionAction) {
   const { collectionId } = action.payload
+  // TODO should we fetch from api instead?
```

Comment:
I think yes

Reply:
should we make a shared method that does the logic below and reuse it in `requestDownloadFavoritedCollectionSaga`?

Reply:
yes i agree, a little worried about the differences with the download favorites i guess? but suppose we can just add an arg there?

Reply:
hmm let's just keep it separate then. No need to overdo reusability